MozillaFoundation/foundation.mozilla.org
# [PNI] Include evaluation object on `ProductPage` revisions

# Description



Fixes #11400 by including the evaluation object on `ProductPage` revisions.

Link to sample test page:
Related PRs/issues: #11400

# Testing

1. Copy prod's db using `inv copy-prod-db` (the migration to fix revisions will be run here)
2. Log in to the CMS
3. Go to a PNI product page
4. Edit the page and save the draft or publish the page
5. See no errors on the live version

# Checklist



**Tests**

**Changes in Models:**

**Documentation:**

**Merge Method**
**💡❗Remember to use squash merge when merging non-feature branches into `main`**
